mdal/mdal_data_model.hpp

```diff
@@ -152,7 +152,7 @@ namespace MDAL
 
       std::string getMetadata( const std::string &key );
       void setMetadata( const std::string &key, const std::string &val );
-      void setMetadata( const Metadata &metadata );
+      void setMetadata( const Metadata &new_metadata );
```

Comment:
why the change of the arg name?

Reply:
because having the same name provokes a compiler warning in windows since the variable hides a class member.

Not a big problem - but it does make sense to avoid the compiler warnings.